sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/api/v2/namespaces/collections/documents/DocumentDeleteResourceIntegrationTest.java

```diff
@@ -0,0 +1,307 @@
+package io.stargate.sgv2.docsapi.api.v2.namespaces.collections.documents;
+
+import static io.restassured.RestAssured.given;
+import static net.javacrumbs.jsonunit.JsonMatchers.jsonEquals;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import io.restassured.RestAssured;
+import io.stargate.sgv2.common.cql.builder.Replication;
+import io.stargate.sgv2.docsapi.config.constants.Constants;
+import io.stargate.sgv2.docsapi.service.ExecutionContext;
+import io.stargate.sgv2.docsapi.service.schema.NamespaceManager;
+import io.stargate.sgv2.docsapi.service.schema.TableManager;
+import io.stargate.sgv2.docsapi.service.write.WriteDocumentsService;
+import io.stargate.sgv2.docsapi.testprofiles.IntegrationTestProfile;
+import java.time.Duration;
+import javax.enterprise.context.control.ActivateRequestContext;
+import javax.inject.Inject;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+@QuarkusTest
+@TestProfile(IntegrationTestProfile.class)
+@ActivateRequestContext
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class DocumentDeleteResourceIntegrationTest {
+
+  public static final String BASE_PATH = "/v2/namespaces/{namespace}/collections/{collection}";
```

Comment:
you can also include `"/v2/namespaces/{namespace}/collections/{collection}/{document-id}"` here so you don't have to repeat it all the time